libstormweaver/src/scripting/luactx.cpp

```diff
@@ -72,7 +72,9 @@ LuaContext::LuaContext(std::shared_ptr<spdlog::logger> logger)
       boost::dll::program_location().parent_path().parent_path().string();
   luaState["package"]["path"] =
       original_package_path + (!original_package_path.empty() ? ";" : "") +
-      fmt::format("{}/scripts/?.lua;{}/scripts_3p/?.lua", base_dir, base_dir);
+      fmt::format("./scripts/?.lua;./scripts_3p/?.lua;{}/scripts/?.lua;{}/"
+                  "scripts_3p/?.lua",
+                  base_dir, base_dir);
```

Comment:
Wouldn't it be cleaner to supply this via a command line parameter for the tests?

Reply:
Yeah, that was what I meant. :)

Reply:
Fixed, I added an `--include` argument.